code-of-conduct.md

```diff
@@ -0,0 +1,53 @@
+# Adoption of document for the OpenData Community
+This a draft until voted on and adopted by the ODC Jedi
+
+# DRAFT Contributor Code of Conduct
+
+As contributors and maintainers in the OpenData Community (ODC), and in the interest of fostering an open and welcoming community, we pledge to respect all people who contribute
+through reporting issues, posting feature requests, updating documentation, submitting pull requests or patches, and other activities.
+
+We are committed to making participation in the ODC community a harassment-free experience for everyone, regardless of level of experience, gender, gender identity and expression, sexual orientation, disability, personal appearance, body size, race, ethnicity, age, religion, or nationality.
+
+## Scope 
+
+This Code of Conduct applies within all project spaces, and it also applies when an individual is representing the project or its community in public spaces. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event.
+
+## ODC Events
+
+ODC events, or events run by ODC sponsor or partner staff, are governed by the same ODC code of conduct. 
+
+## Our Standards
+
+Examples of behavior that contributes to a positive environment include:
+
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Focusing on what is best not just for us as individuals, but for the overall community
+* Showing empathy towards other community members
+* Do not misrepresent your identity in a deceptive or harmful way. This includes creating fake profiles and attempts to impersonate an individual, group, or organization
+* Do not engage in activities that could damage or compromise the security of an account, network, or system
+* Do not use or attempt to use ODC to promote, coordinate, or execute financial scams
+* Do not engage in activities that fraudulently generate a profit at the expense of others
+
+Examples of unacceptable behavior include:
+
+* The use of sexualized language or imagery and unwelcome sexual attention or advances
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email address, without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a professional setting
+* Sending unsolicited bulk messages (or spam) to others
+
+Project maintainers have the right and responsibility to remove, edit, or reject comments, commits, code, wiki edits, issues, and other contributions that are not aligned to this Code of Conduct. 
+
```

Comment:
felt that maybe here we have a sharp turn . Are we talking about "odc jedis" as projects maintainers? I got confused if thats the case or we are talking of future Sandbox project leads  .

Reply:
I should change that to project lead or sandbox team lead. 

See roles https://github.com/OpenDataforWeb3/governance/blob/2aace55a20d8c964efc4cdfce8fb196151508185/governance.md#roles